components/script/dom/htmlanchorelement.rs

```diff
@@ -108,6 +110,101 @@ impl HTMLAnchorElementMethods for HTMLAnchorElement {
 
     // https://html.spec.whatwg.org/multipage/#dom-a-shape
     make_setter!(SetShape, "shape");
+
+    // https://html.spec.whatwg.org/multipage/semantics.html#dom-hyperlink-hash
```

Comment:
This should be `https://html.spec.whatwg.org/multipage/#dom-hyperlink-hash`

You should run `./mach test-tidy` to see the errors that are occurring